src/app/authentication.service.ts

```diff
@@ -0,0 +1,9 @@
+import { Injectable } from '@angular/core';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class AuthenticationService {
+
```

Comment:
Perhaps you can add here a variable `loggedIn = false;` and the method `isAuthenticated()` that returns the value of the `loggedIn` variable.